src/module.c

```diff
@@ -11224,8 +11224,8 @@ static void moduleScanKeyHashtableCallback(void *privdata, void *entry) {
         key = node->ele;
         value = createStringObjectFromLongDouble(node->score, 0);
     } else if (o->type == OBJ_HASH) {
-        key = hashTypeEntryGetField(entry);
-        sds val = hashTypeEntryGetValue(entry);
+        key = entryGetField(entry);
```

Comment:
How does this work with Vector search? I don't really know much about how vector search actually get's the notification to ingest the key and pull the data out. There is some async component there.

Reply:
Well I did not change any logic here. the same was in 8.1, so if it worked there it should in this case